src/week-view.c

```diff
@@ -67,6 +69,14 @@ enum {
 	PROP_VSCROLL_POLICY,
 };
 
+GdkRGBA *color_bg;
+GdkRGBA *color_header_bg;
+GdkRGBA *color_fg;
+GdkRGBA *color_fg_50;
+GdkRGBA *color_header_divider;
+GdkRGBA *color_current_time_marker;
+GdkRGBA *color_current_day;
```

Comment:
In general global variables should be avoided - I suggest making these static or putting them inside a static struct named e.g. `palette` or `colors`. Furthermore, there's no need for these to be pointers which have to be malloc'd later, they can be regular variables. Then `allocate_colors` is unnecessary

Reply:
i agree, thank you for the hint how to use an attribute here.